starport/services/chain/plugin-launchpad.go

```diff
@@ -17,21 +19,14 @@ import (
 )
 
 type launchpadPlugin struct {
-	app App
-	cmd chaincmd.ChainCmd
+	app   App
+	chain *Chain
```

Comment:
The plugin is already an attribute of the `chain` so I don't think it's a good pattern to have.
But it's ok to let it now. In https://github.com/tendermint/starport/pull/615, I do a refactoring and let `chain` providing the command instance when calling functions from the plugin

Reply:
Yes, this is a circular dependency right there. :smiling_imp: 

I was planning to get rid of it and in-fact improve the overall plugin architecture within https://github.com/tendermint/starport/issues/608. Let's coordinate on this to not get overlapped with each other's code. :keyboard: :computer: